unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -1006,6 +1035,46 @@ renderDoc ppe width rt codebase r = do
       fmap (DD.amap (const ())) <$> Codebase.runTransaction codebase (Codebase.getTypeDeclaration codebase r)
     decls _ = pure Nothing
 
+-- | Fetch the docs associated with the given name.
+-- Returns all references with a Doc type which are at the name provided, or at '<name>.doc'.
+docsForTermName ::
+  Codebase IO Symbol Ann ->
+  NameSearch Sqlite.Transaction ->
+  Name ->
+  IO [TermReference]
+docsForTermName codebase (NameSearch {termSearch}) name = do
+  let potentialDocNames = [name, name Cons.:> "doc"]
+  Codebase.runTransaction codebase do
+    refs <-
+      potentialDocNames & foldMapM \name ->
+        -- TODO: Should replace this with an exact name lookup.
```

Comment:
We should do this in the future, but for now we retain the existing functionality.